.ci/build-linux.sh

```diff
@@ -38,15 +38,15 @@ fi
 
 export CFLAGS="$CFLAGS -fuse-ld=${LINKER}"
 
-cmake ..                                               \
-    -DCMAKE_INSTALL_PREFIX=/usr                        \
-    -DBUILD_LLVM_SUBMODULE=OFF -DUSE_COTIRE=OFF        \
-    -DLLVM_DIR=llvmlibs/lib/cmake/llvm/                \
-    -DUSE_NATIVE_INSTRUCTIONS=OFF                      \
-    -DCMAKE_C_FLAGS="$CFLAGS"                          \
-    -DCMAKE_CXX_FLAGS="$CFLAGS"                        \
-    -DCMAKE_AR="$AR"                                   \
-    -DCMAKE_RANLIB="$RANLIB"                           \
+cmake ..                                                      \
+    -DCMAKE_INSTALL_PREFIX=/usr                               \
+    -DBUILD_LLVM_SUBMODULE=OFF -DUSE_PRECOMPILED_HEADERS=OFF  \
```

Comment:
can't you just put this in a new line instead?

Reply:
Yes, done